src/_includes/macros/util/format/datetime.html

```diff
@@ -0,0 +1,77 @@
+{# ==========================================================================
+
+   format_time()
+
+   ==========================================================================
+
+   Description:
+
+   Format a time for displaying in the format of:
+
+    HH:MM AM ZZZ
+
+   When given:
+
+   datetime: A datetime object.
+
+   timezone (optional): Whether to include the timezone or not.
+                        Provide '' to default to 'US/Eastern',
```

Comment:
Default is 'America/New_York` on https://github.com/cfpb/cfgov-refresh/pull/925/files#diff-dcbab91c61a5b3f8b4fee5b18310b05eR22 these should be the same.